k8s/prod/common/ccpay/ccpay-payment-app.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccpay-payment-app
+  namespace: fees-pay
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ccpay-payment-api
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: payment-api
+    version: 0.1.0
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      image: hmctspublic.azurecr.io/payment/api:prod-a4872e81
+      ingressHost: payment-api-prod.service.core-compute-aat.internal
+      applicationInsightsInstrumentKey: "52125c3c-ea7b-4954-a872-fe6987780a88"
```

Comment:
Are you sure? ccpay-prod and ccpay-appinsights-prod have different keys from this one.